fs/move.py

```diff
@@ -57,6 +58,8 @@ def move_file(
         dst_path (str): Path to a file on ``dst_fs``.
         preserve_time (bool): If `True`, try to preserve mtime of the
             resources (defaults to `False`).
+        cleanup_dest_on_error (bool): If `True`, tries to delete the file copied to
+            dst_fs if deleting the file from src_fs fails.
```

Comment:
This docstring should probably also mention the default?

Reply:
Agreed. And it should be `cleanup_dst_on_error`, shouldn't it?